### ISSUE TITLE
(feature request) second screen in single screen mode

### ISSUE THREAD

**Issue by Moodie92** [citra-emu/citra#4381]
it will be a really good addition to assign a button to show the second screen when using the single screen mode , just like cemu emulator when pressing TAB to show the controller screen

**Comment by SachinVin** [comment]
You can already do that by enabling swap screen. If nothing has change since the last time i used citra, the shortcut should be F9

**Comment by legoj15** [comment]
Yes, but that is a toggle, not a button hold, like how CEMU shows the DRC screen.

**Comment by PandarinDev** [comment]
AFAIK currently there is no way of handling "key down" and "key release" events for shortcuts that this feature would require. It would require changes in the input system, so I'm not sure the E-easy tag is correct here. Of course take my opinion with a grain of salt, as I'm new to the development of Citra.

In the meantime toggling screens with F9 seems a reasonable workaround to me.

**Comment by JohelEGP** [comment]
#5119 is better in details for those who haven't used Cemu.

**Comment by SleepingSnakezzz** [comment]
Closing in favour of https://github.com/citra-emu/citra/issues/5119 and https://github.com/citra-emu/citra/issues/4932
